proposals/0029-cooperative-vector.md

```diff
@@ -528,19 +607,24 @@ typedef enum D3D12_FEATURE {
     D3D12_FEATURE_COOPERATIVE_VECTOR;
 };
 
+// This is designed to match the ComponentType enum values but omits data 
+// types that are not currently specified to work with this API. The names are chosen
+// to more closely match those used by HLSL developers, as opposed to the ComponentType 
+// names that align with LLVM IR.
+
 typedef enum D3D12_COOPERATIVE_VECTOR_DATATYPE {
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_FLOAT16      = 0,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_FLOAT32      = 1,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_UINT8        = 2,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_UINT16       = 3,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_UINT32       = 4,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_SINT8        = 5,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_SINT16       = 6,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_SINT32       = 7,    
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_SINT8_PACKED = 8,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_UINT8_PACKED = 9,
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_FLOAT_E4M3   = 10,      // FP8: 1 sign bit, 4 exp bits, 3 mantissa bits
-    D3D12_COOPERATIVE_VECTOR_DATATYPE_FLOAT_E5M2   = 11       // FP8: 1 sign bit, 5 exp bits, 2 mantissa bits
+  D3D12_COOPERATIVE_VECTOR_DATATYPE_SINT16          =  2, // ComponentType::I16
```

Comment:
Can we rename D3D12_COOPERATIVE_VECTOR_* to D3D12_LINEAR_ALGEBRA_*?  (can be separate PR if that's more appropriate).  My assumption is in HLSL the name space is dx::linalg?

Even for the tier.  I'd imagine if we add matrix*matrix support it would be TIER_2_0.   And if we add capabilities specific to coop vec in the future, they would just be independent caps (and/or also part of TIER_2_0).

Reply:
Let's do that in a separate PR.